Jenkinsfile

```diff
@@ -63,52 +62,27 @@ node ('master') {
             env.ISOLATION_ID = sh(returnStdout: true, script: 'printf $BUILD_TAG | sha256sum | cut -c1-64').trim()
             env.COMPOSE_PROJECT_NAME = sh(returnStdout: true, script: 'printf $BUILD_TAG | sha256sum | cut -c1-64').trim()
 
-            stage("Build Lint Dependencies") {
-                sh 'docker-compose up --abort-on-container-exit --build --force-recreate --renew-anon-volumes --exit-code-from poet-common poet-common'
-                sh 'docker-compose down'
-            }
-
-            stage("Run Lint") {
```

Comment:
This should be replaced with a rust lint check.

Reply:
> Appreciate the work that has gone into more granular commits thus far. Still, though it is easier than it was, the commit messages do not attempt to explain the reasoning for the changes, and it can not be easily inferred.
> 
> For example, in this commit:
> 
> commit [f6145a2](https://github.com/hyperledger/sawtooth-poet/commit/f6145a2ac23cd8c2c60e3db197024e94b6e5ab4b)
> Author: Rajeev Ranjan [rajeev2.ranjan@intel.com](mailto:rajeev2.ranjan@intel.com)
> Date: Thu Nov 15 14:11:53 2018 +0530
> 
> ```
> UTs for ConsensusStateStore & lmdb
> 
> Updations in lmdb.rs. Implement new uts in ConsensusStateStore.
> 
> Signed-off-by: Rajeev Ranjan <rajeev2.ranjan@intel.com>
> ```
> It is not clear the purpose of these changes. Since this change in particular is a change to a file copied from sawtooth-core, I'm curious about the reasoning. Good explanation in the commit message would help. (See https://chris.beams.io/posts/git-commit/)

Will edit the commit message in the commit where this file got introduced. This file was used from sawtooth-core since lmdb was not exposed via sawtooth-sdk. lmdb file is used in consensus state store

Reply:
> This should be replaced with a rust lint check.

Will add Rust Lint and Rust fmt as separate commits

Reply:
@vaporos Fixed clippy errors and rustfmt and added these checks in Jenkins build